src/index.ts

```diff
@@ -30,12 +29,21 @@ configure({
 });
 
 const start = async (): Promise<void> => {
+    let appConfig;
+    if (process.env.NODE_ENV === "development" && fs.existsSync("../app-config.json")) {
+        appConfig = JSON.parse(fs.readFileSync("../app-config.json", 'utf8'));
+    } else if (fs.existsSync("app-config.json")) {
+        appConfig = JSON.parse(fs.readFileSync("app-config.json", 'utf8'));
+    }
+
```

Comment:
This feels a bit ad hoc and fragile. Proposal: Use yargs/argparse/whatever-you-like and add a (required) option `--config-file CONFIG.json`. What do you think?

Reply:
Yeah, looks good to me. As I have no strong opinion on which is better (only used ```vorpal``` in the past and it's way to much for what we need), I'm adding yargs.

```
Options:
  --help        Show help                                              [boolean]
  --version     Show version number                                    [boolean]
  -c, --config                                               [string] [required]
```